### ISSUE TITLE
Another Ruby 2.7 warning

### ISSUE THREAD

**Issue by jimwise** [chef/chef#9879]
https://github.com/chef/chef/blob/b84d384a058d2159c610771beccfe7728c28a688/lib/chef/resource/hostname.rb#L152

Please add a `**` before `{ returns: [0, 1] }`.

**Comment by jimwise** [comment]
Failure mode:
```
/opt/chef/embedded/lib/ruby/gems/2.7.0/gems/chef-16.1.0/lib/chef/resource/hostname.rb:152: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/opt/chef/embedded/lib/ruby/gems/2.7.0/gems/chef-16.1.0/lib/chef/mixin/shell_out.rb:54: warning: The called method `shell_out!' is defined here
```

**Comment by btm** [comment]
Fixed in #9883
